swupd/create_manifests.go

```diff
@@ -177,6 +192,70 @@ func processBundles(ui UpdateInfo, c config) ([]*Manifest, error) {
 	return newManifests, nil
 }
 
+func appendUniqueFiles(appendTo *Manifest, appendFrom *Manifest) {
+	for _, f := range appendFrom.Files {
+		if f.findFileNameInSlice(appendTo.Files) == nil {
+			if f.Name == indexBundle {
+				// this is generated new each time
+				continue
+			}
+			appendTo.Files = append(appendTo.Files, f)
+		}
+	}
+}
+
+func writeMansPopulateMoM(newManifests []*Manifest, newMoM *Manifest, out string) (*Manifest, error) {
+	var newFull *Manifest
+	var err error
+	// write manifests then add them to the MoM
+	for _, bMan := range newManifests {
+		// handle the full manifest last
```

Comment:
We are not "handling" the full manifest in this function now.

I suggest a comment on this function to say it returns the "full" manifest.

I think this function ought to both take and return newMom, or perhaps be a method on Manifest.